src/shared/components/ActionBar/ActionBar.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import {
+  StyledActionBarContainer,
+  StyledInfoContainer,
+  StyledPrimaryText,
+  StyledSecondaryText,
+  StyledDetailsTextContainer,
+  StyledButtonsContainer,
+  StyledSecondaryButton,
+} from './ActionBar.style'
+import { Button, Icon } from '@/shared/components'
+import type { IconType } from '../Icon'
+
+export type ActionBarProps = {
+  primaryText?: string
+  secondaryText?: string
+  primaryButtonText?: string
+  detailsText?: string
+  detailsTextIcon: IconType
+  secondaryButtonText?: string
+  secondaryButtonIcon?: IconType
+  onClickPrimaryButton?: (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void
+  onClickSecondaryButton?: (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void
+}
+
+const ActionBar: React.FC<ActionBarProps> = ({
```

Comment:
This works fine as a generic action bar but I think it would be beneficial to also add a component that's specifically targeting the transaction use case, as we will be reusing that for few views. It would use this one inside it. So for example, instead of `primaryText`, it would accept the number of joy and add the `Fee: ...` itself, it'd specify the `secondaryText` internally, etc. The props that aren't consistent will still need to be passed.

This way we will be able to avoid some duplication across views and easier time if we want to update the `secondaryText` for example.

For the text, let's use `Every change in the blockchain state requires paying a nominal fee.`